wikimedia/wikipedia-ios
# Diffs - Update thanks icon

**Phabricator:** https://phabricator.wikimedia.org/T335579 (partially)

### Notes
Updates thanks icon to use new design.

### Test Steps
1. Open a diff comparison view
2. Confirm the new thanks icon displays in toolbar
3. Tap to thank, confirm filled variant is displayed

### Screenshots/Videos
![thanks-diff](https://github.com/wikimedia/wikipedia-ios/assets/5652327/0db6adfc-c657-4aa1-b161-31e1b2d8a18d)
